doc/api/crypto.md

```diff
@@ -1241,18 +1241,14 @@ input.on('readable', () => {
 added: v0.1.92
 -->
 
-Creates and returns a `Sign` object that uses the given `algorithm`. On
-recent OpenSSL releases, `openssl list-public-key-algorithms` will
-display the available signing algorithms. One example is `'RSA-SHA256'`.
+Creates and returns a `Sign` object that uses the given `algorithm`. For available signing algorithms, please reference `crypto.getHashes()`.
```

Comment:
Using `reference` makes it sound like you should refer to the documentation for `crypto.getHashes()` but what  is really meant is to run it to obtain the array of available algorithms, right? If so, instead of this:

> For available signing algorithms, please reference `crypto.getHashes()`.

...I'd prefer something like:

> Use [`crypto.getHashes()`][] to obtain an array of names of the available signing algorithms.